otcdocstheme/theme/otcdocs/header.html

```diff
@@ -14,7 +14,7 @@
         <span class="icon-bar"></span>
       </button>
       <div class="brand-wrapper">
-        <a class="navbar-brand" href="https://cloud.telekom.de/en/infrastructure/open-telekom-cloud/"></a>
+        <a class="navbar-brand" href="https://open-telekom-cloud.com/de"></a>
```

Comment:
Should not have a "de" in the end: https://open-telekom-cloud.com/

Reply:
I have to create another correction, before my push. Link is also fine now.